apps/platform-app/src/app/[locale]/(profile)/_actions/queries/findLatestRejectionReason.ts

```diff
@@ -0,0 +1,12 @@
+'use server'
+import { getProfileById } from '@/backend/profile/profile.service'
+import { getRejectionReason } from '@/backend/profile/rejection.service'
+import { withSentry } from '@/utils/errHandling'
+import { cache } from 'react'
+
+export const findLatestRejectionReason = cache(
```

Comment:
**suggestion**: Consider adding a return type to the `findLatestRejectionReason` function for better type safety.